frontend/packages/dev-console/integration-tests/features/monitoring/monitoring-Page-Smoke-Metrics-ODC-4116.feature

```diff
@@ -0,0 +1,165 @@
+Feature: Metrics Tab on Monitoring Page
+    As a user, I should be able to run queries to check the usage
+
+
+Background: 
+    Given user is at Developer perspective
+
+
+@smoke, @regression
+Scenario: CPU Usage Query on Metrics tab: MH-01-TC04
+    Given user is on Monitoring page
```

Comment:
Add the given statement 
"And open project namespace "project name"

Reply:
Added